lib/ocpp/v16/charge_point_impl.cpp

```diff
@@ -2371,11 +2373,21 @@ void ChargePointImpl::log_status_notification(UploadLogStatusEnumType status, in
 }
 
 void ChargePointImpl::signed_firmware_update_status_notification(FirmwareStatusEnumType status, int requestId) {
-    EVLOG_debug << "Sending FirmwareUpdateStatusNotification";
+    EVLOG_debug << "Sending FirmwareUpdateStatusNotification with status"
+                << conversions::firmware_status_enum_type_to_string(status);
```

Comment:
Isn't there a generic debug-log in `this->send()` already? :thinking:

Reply:
Hmm which one do you mean? Could be that the websocket has some generic debug log. But for me the crucial aspect was that the logs actually reveal which (and in which order) firmware status updates (meaning the value) are sent and I don't think any existing log prints those (so I left this in case the firmware update make trouble again in the future)